src/audio/sr_audio/sr_audio.c

```diff
@@ -0,0 +1,720 @@
+// Copyright(c) 2021 Sound Research Corporation. All rights reserved.
+#include <sof/audio/component.h>
+#include <sof/audio/buffer.h>
+#include <sof/audio/sr_audio/sr_audio.h>
+#include <sof/audio/format.h>
+#include <sof/audio/pipeline.h>
+#include <sof/common.h>
+#include <sof/debug/panic.h>
+#include <sof/ipc/msg.h>
+#include <sof/lib/alloc.h>
+#include <sof/lib/memory.h>
+#include <sof/lib/uuid.h>
+#include <sof/list.h>
+#include <sof/platform.h>
+#include <sof/string.h>
+#include <sof/ut.h>
+#include <sof/trace/trace.h>
+#include <ipc/control.h>
+#include <ipc/stream.h>
+#include <ipc/topology.h>
+#include <user/trace.h>
+#include <errno.h>
+#include <stddef.h>
+#include <stdint.h>
+
+//#define _ADD_TEST_CLICK_AT48
```

Comment:
Debug option ? Best to comment how it should be used by integrators.

Reply:
I don't believe it should be used by integrators at all. This is our internal test macro and its name basically tells you what it does, so the comment would be 'Add a test click and 48th sample position'. Anyway, will add this info to the code.